src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -139,12 +139,13 @@ internal void SetBuffer(byte[] buffer, int offset, int count)
         {
             _bufferedContent = new MemoryStream(buffer, offset, count, writable: false, publiclyVisible: true);
         }
-        
+
         internal bool TryGetBuffer(out ArraySegment<byte> buffer)
         {
+            buffer = default(ArraySegment<byte>);
```

Comment:
This PR compiles fine in CI, so that line isn't needed and I took it out in the 2nd commit.